### ISSUE TITLE
Link MetaCpan to github

### ISSUE THREAD

**Issue by m0veax** [itnode/DBIx-Class-Elasticsearch#6]
META_MERGE        => {
         resources => {
             repository  =>  'https://github.com/interchange/interchange6-schema.git',
         }
}

**Comment by karenetheridge** [comment]
No, it should be:

```
    META_MERGE => {
        'meta-spec' => { version => 2 },
        dynamic_config => 0,            # unless you know you need it to be true
        resources => {
            repository => {
                url => 'https://github.com/$user/$repo.git',
                web => 'https://github.com/$user/$repo'
                type => 'git',
            },
            bugtracker => {
                mailto => 'bug-DBIx-Class-ElasticSearch@rt.cpan.org',
                web => 'https://rt.cpan.org/Public/Dist/Display.html?Name=DBIx-Class-ElasticSearch',
            },
        },
    },
```

Also, if you're going to use RT as your bugtracker, please disable github 'issues'.
Or, if you want github to be the primary issue tracker, provide that URL in the bugtracker resources.

**Comment by m0veax** [comment]
Thanks for your Feedback :)

We want to use Github and will change that in the next Release. I have to figure out how exactly that works

**Comment by m0veax** [comment]
fixed by c3d66c46e83e6b407a34285bdc25118728df0a80
